src/widgetastic_patternfly4/card.py

```diff
@@ -0,0 +1,59 @@
+from widgetastic.log import call_sig
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import ClickableMixin
+from widgetastic.widget import ParametrizedView, ParametrizedViewRequest, GenericLocatorWidget
+from widgetastic.widget import Widget, View
+from widgetastic.xpath import quote
+from itertools import count
+
+class Card(GenericLocatorWidget):
+
+    def __init__(self, parent, locator=None, logger=None, **kwargs):
+        View.__init__(self, parent, logger=logger, **kwargs)
+        self.locator = locator or ".//article[contains(@class, 'pf-c-card')]"
+    ROOT = ParametrizedLocator("{@locator}")
+
+
+class CardForCardGroup(ParametrizedView ,Card):
```

Comment:
Why do you inherit `Card` here?

Reply:
cards themselfs are components that can be used outside of collections, but they still have behaviours we miht want to add (like the checked state in genera, or action groups